pkg/containerd/events.go

```diff
@@ -191,6 +217,30 @@ func (e *EventMonitor) createPortMapping(ctx context.Context, containerID string
 	return portMap, nil
 }
 
+func updateListener(ctx context.Context, portMappings nat.PortMap, action func(context.Context, net.IP, int) error) {
+	for _, portBindings := range portMappings {
+		for _, portBinding := range portBindings {
+			port, err := strconv.Atoi(portBinding.HostPort)
+			if err != nil {
+				log.Errorf("port conversion for [%s] error: %v", portBinding.HostPort, err)
+
+				continue
+			}
+
+			// We always need to use INADDR_ANY here since any other addresses used here
+			// can cause a wrong entry in the IP Table and will not be routable.
```

Comment:
```suggestion
			// can cause a wrong entry in iptables and will not be routable.
```
`iptables` is a specific command, and _IP Table_ isn't a thing.